src/fideslang/validation.py

```diff
@@ -93,3 +94,46 @@ def check_valid_country_code(country_code_list: List) -> List:
                     )
                 )
     return country_code_list
+
+
+def parse_data_type_string(type_string: Optional[str]) -> Tuple[Optional[str], bool]:
+    """Parse the data type string. Arrays are expressed in the form 'type[]'.
+
+    e.g.
+    - 'string' -> ('string', false)
+    - 'string[]' -> ('string', true)
+
+    These data_types are for use in DatasetField.fides_meta.
+    """
+    if not type_string:
+        return None, False
+    idx = type_string.find("[]")
+    if idx == -1:
+        return type_string, False
+    return type_string[:idx], True
+
+
+# Data types that Fides is currently configured to handle
+data_type_names: Set[str] = {
```

Comment:
```suggestion
DATA_TYPE_NAMES: Set[str] = {
```

Nit: since its a global, I think it should be upper-cased so I know immediately where to look for it